src/components/form.jsx

```diff
@@ -0,0 +1,36 @@
+import { useForm } from "react-hook-form";
+// the Form component take as argument the children elements, an onSubmit function and atributes for a form tag.
+const Form = ({ children, onSubmit, ...htmlFormAttributes }) => (
```

Comment:
1) Updating this to use named exports
2) Adding a `className` prop
3) We'll probably be using the "rest" props all over the place, so no need to name them something unique each time

```suggestion
export const Form = ({ className, children, onSubmit, ...rest }) => {
```